### ISSUE TITLE
Climatisation state

### ISSUE THREAD

**Issue by tieskuh** [cyr-ius/hass-audiconnect#75]
In one of the previous versions there was a sensor 'Climatistation State'. This reflected the actual state. If could have the values:

- heating
- cooling
- off

So it should be a sensor, not a binary_sensor.

I guess it is this function, looking at the diagnostics file.

"getClimatisationStatus": {
                  "id": "getClimatisationStatus",
                  "scopes": [
                    "climatisation"
                  ]
                },

[config_entry-audiconnect-4b46c0432e37e4a752376be96c057293.json](https://github.com/user-attachments/files/15761350/config_entry-audiconnect-4b46c0432e37e4a752376be96c057293.json)

Is is possible to add the sensor?

**Comment by cyr-ius** [comment]
unfortunately the only information we can provide is the following:
```
          "climatisation": {
            "climatisationSettings": {
              "value": {
                "carCapturedTimestamp": "2024-06-10T09:47:00Z",
                "targetTemperature_C": 20,
                "targetTemperature_F": 68,
                "climatisationWithoutExternalPower": true,
                "climatizationAtUnlock": false,
                "windowHeatingEnabled": true,
                "zoneFrontLeftEnabled": false,
                "zoneFrontRightEnabled": false
              }
            },
            "climatisationStatus": {
              "value": {
                "carCapturedTimestamp": "2024-06-10T09:47:05Z",
                "remainingClimatisationTime_min": -256,
                "climatisationState": "off"
              }
            },
            "windowHeatingStatus": {
              "value": {
                "windowHeatingStatus": [
                  {
                    "windowLocation": "front",
                    "windowHeatingState": "invalid"
                  },
                  {
                    "windowLocation": "rear",
                    "windowHeatingState": "invalid"
                  }
                ]
              }
            }
          },
```

You should turn on the vehicle's air conditioning and see that the diagnostic file contains additional values

The information collected begins with the entry "async_get_selectivestatus": { xxxxxxxxxxx.........}

**Comment by tieskuh** [comment]
I think it is this one:
```"climatisationState": "off"```


When I start the climater and download diagnostics, it shows:

```
"climatisationStatus": {
              "value": {
                "carCapturedTimestamp": "2024-06-10T11:19:35Z",
                "remainingClimatisationTime_min": 29,
                "climatisationState": "heating"
              }
            },
```


The value can be: 

- heating
- cooling
- off

**Comment by cyr-ius** [comment]
OK , fix to 2.1.0

**Comment by tieskuh** [comment]
Fixed! Thanks a lot!!
